music_this_week_app/backend/spotifyHandler.py

```diff
@@ -31,8 +40,18 @@ def filter_list_of_artists(self, unfiltered_artists):
         :return artistURIs: list of strings of verified Spotify artist identifiers
         """
 
-        artistURIs = [self.filter_artist(a) for a in unfiltered_artists]
+        artistURIs = []
+        for artist_name in unfiltered_artists:
+            try:   
+                artist_uri = self.filter_artist(artist_name)
+                artistURIs.append(artist_uri)
+                self.artists_to_save[artist_uri] = artist_name
+            except ValueError:
```

Comment:
this was the least confusing way I could thing of conveying that the artist already existed in DB.

Value Error is obviously not an appropriate choice of Exception